main.go

```diff
@@ -130,93 +121,36 @@ func main() {
 
 	cacherClient, err := client.New(*facility)
 	if err != nil {
-		panic(err)
+		logger.Fatal(err, "Failed to create the cacher client")
 	}
-	go func() {
-		c := time.Tick(15 * time.Second)
-		for range c {
-			// Get All hardware as a proxy for a healthcheck
-			// TODO (patrickdevivo) until Cacher gets a proper healthcheck RPC
-			// a la https://github.com/grpc/grpc/blob/master/doc/health-checking.md
-			// this will have to do.
-			// Note that we don't do anything with the stream (we don't read from it)
-			var isCacherAvailableTemp bool
-			ctx, cancel := context.WithCancel(context.Background())
-			_, err := cacherClient.All(ctx, &cacher.Empty{})
-			if err == nil {
-				isCacherAvailableTemp = true
-			}
-			cancel()
-
-			isCacherAvailableMu.Lock()
-			isCacherAvailable = isCacherAvailableTemp
-			isCacherAvailableMu.Unlock()
-
-			logger.With("status", isCacherAvailableTemp).Debug("tick")
-		}
-
-	}()
```

Comment:
Why was this removed?

Reply:
I didn't see any use case for it so I removed to keep the code simple.
could you please brief me about the use case for it?

Reply:
Revert back.